server/models/retrieveEmployees.js

```diff
@@ -0,0 +1,19 @@
+import pool from '../config/dbConnect';
+
+ 
+     const adminRetrieve = async(req, res)=> {
+    const findAllEmployees = 'SELECT * FROM  users';
```

Comment:
there is a file `server/models/queries.js` which holds all queries if you have some time you can put all this query inside it and then import it here. I think it would be better if we keep things organised.